test/ngIntegration/tests/caseFieldCollectionsPermissions.js

```diff
@@ -0,0 +1,124 @@
+;
+
+
+const assert = require('assert');
+
+const MockApp = require('../../nodeMock/app');
+const { browser } = require('protractor');
+const BrowserUtil = require('../util/browserUtil');
+const BrowserWaits = require('../../e2e/support/customWaits');
+
+const headerPage = require('../../e2e/features/pageObjects/headerPage');
+const exuiTestCaseType = require('../../nodeMock/ccd/solicitorCreate/exuiTestCaseType');
+const { createNonNullExpression } = require('typescript');
+
+describe.skip('CCD-CASE-UI-TOOLKIT collection field permissions', function () {
+
+    beforeEach(async function (done) {
+        MockApp.init();
+        done();
+    });
+    afterEach(async function (done) {
+        await MockApp.stopServer();
+        done();
+    });
+
+    const addNewButton = element(by.xpath("//div[@id = 'textCollection']//button[text() = 'Add new']"));
+    const removeButton = element(by.xpath("//div[@id = 'textCollection']//button[text() = 'Remove']"));
+    
+    const roles = ["caseworker-divorce-financialremedy-solicitor"];
+
+    it('display_context_parameter null', async function () {
+        setUpcaseConfig(null); 
+
+        await MockApp.startServer();
+        await BrowserUtil.browserInitWithAuth(roles);
+        await headerPage.isTabPresent('Case list');
+        await browser.get("http://localhost:4200/cases/case-create/exui/casetype_1/submitDraft/page1");
```

Comment:
It's probably worth placing http://localhost:4200 in a constant somewhere.

Reply:
updated. replaced with value from config. in all places in this test file.